src/Compactable.ts

```diff
@@ -32,21 +32,12 @@ import {
 } from './Functor'
 import { HKT, Kind, Kind2, Kind3, Kind4, URIS, URIS2, URIS3, URIS4 } from './HKT'
 import { getLeft, getRight, Option } from './Option'
+import { separated, Separated } from './Separated'
 
 // -------------------------------------------------------------------------------------
 // model
 // -------------------------------------------------------------------------------------
 
-/**
- * A `Separated` type which holds `left` and `right` parts.
- *
- * @since 2.0.0
- */
-export interface Separated<A, B> {
```

Comment:
We can't remove this interface, would be a breaking change. My suggestion is to re-export from the `Separated` module